spec/models/card_spec.rb

```diff
@@ -14,12 +14,23 @@
     expect(card.check_translation("ifdnq02")).to be false
   end
 
-  it "#review_date" do
-    expect(card.review_date).to eq(Date.today + 3.days)
+  it "card after true translation" do
+    Timecop.freeze(Time.now)
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.